### ISSUE TITLE
Issues with Julia forking

### ISSUE THREAD

**Issue by fingolfin** [oscar-system/GAPJulia#54]
Right now, JuliaInterface contains this hack in `JuliaImportPackage`:
```
        #Try to compile package by loading it into external Julia
        #Get Julia path
        julia_path := ConvertedFromJulia( JuliaEvalString( "Sys.BINDIR" ) );
        Exec( Concatenation( julia_path, "/../bin/julia -e \"", callstring,
                  "\"" ) );
```
The reason for this (as I just learned from @sebasguts ) is that otherwise, Julia tries to compile the given package itself, which requires a fork; but that fails with some error (apparently in `libuv`)

For now, this workaround is good enough, but it'd be useful to figure out what exactly the problem is, and how to fix it properly.

**Comment by sebasguts** [comment]
This bug seems to be gone. I think we could savely remove this whole Exec stuff. This also applies to #38

**Comment by fingolfin** [comment]
Resolved by PR #55
